principles/enterprise-architecture-principles.md

```diff
@@ -5,40 +5,23 @@ expires: 2020-01-01
 
 # Enterprise Architecture Principles
 
-(Draft, structure only)
-
-These principles are stated in no particular order, and are *always* open to debate.
-Architecture and design is a trade-off between competing pressures - these principles are
-meant to help you decide which trade-offs are acceptable.
-
-They are guidance, not The Law - there will always be edge cases, but you should
-expect to be challenged if you go your own way. The principles are to guide future
-and current architecture - use your judgement, but in general, follow the principle of
-'reuse, before buy, before build'.
-
-They seek to inform those seeking to invest in solutions for DfE of the architectural principles
-that have been agreed, which will govern data, technology and applications investments.
-
-The DfE Architecture Review Board will review and agree these principles on a 6 monthly basis.
-
-## Foundation Principles
-These are the foundation principles that guide the way in which DfE investments
-in data, technology and applications will be undertaken.
-
-### 1. User needs have primacy
-_Statement_ - The direction set out by Digital, Data and Technology (DDaT) directorates, and the path which it pursues, must always seek to solve business problems for DfE’s users – i.e. schools, teachers, providers, students, apprentices, industry, communities, partners and staff. The needs of these groups, and the DfE business areas that serve them, take primacy over other actors and constraints.
-
-_Rationale_ - Identifying, defining and subsequently solving the right user problem is key. Using architecture techniques and tools, including frameworks and roadmaps, will demonstrate clear linkage to business priorities, drivers and associated problems. DDaT will deliver the outcomes that represent user needs as capabilities for DfE business areas.
-
-_Implications_ - DDaT will pursue solutions in accordance with real user needs assuring that the path taken produces the right outcome. As a result of this principle:
-*	DDaT must cultivate a deep understanding of business area strategy, direction, aspirations, challenges and needs using a portfolio, rather than a project, view.
-*	DDaT must demonstrate understanding of user needs by using artefacts such as user stories and user journeys, to identify what is required of both DfE business and DDaT teams.
-*	DDaT must ensure that all layers of data, application and technology services and solutions are focused on delivering user needs.
-*	Solutions delivered by DDaT projects resulting in new or amended services will demonstrate user-centred outcomes.
-
-### 2. Heading
-_Statement_ - 
-
-_Rationale_ -  
-
-_Implications_ -
+Principles are general rules and guidelines, intended to be enduring and seldom amended, that inform and support the way in which the DfE fulfils its vision and principles. They reflect a level of consensus across the enterprise and embody the spirit and thinking of the enterprise architecture.
+
```

Comment:
Merge in some of what I originally wrote? Feels a bit hard-edged as is, inflexible, unsupportive and is likely to cause some stirs.